test/AppNotificationTests/AppNotificationTests.vcxproj

```diff
@@ -39,7 +39,7 @@
   <PropertyGroup>
     <ConfigurationType>DynamicLibrary</ConfigurationType>
     <UseOfMfc>false</UseOfMfc>
-    <PlatformToolset>v142</PlatformToolset>
+    <PlatformToolset>v143</PlatformToolset>
```

Comment:
do you need this change?

Reply:
technically no, but it gets pretty anoying fast when you open a clean solution in VS2022. It looks like Howard missind these two projects when he bumped the solution to v143 in: https://github.com/microsoft/WindowsAppSDK/commit/e46df570bb8cf4a1bf2692b15461584a18b925f1